tools/gulp/tasks/material-release.ts

```diff
@@ -32,7 +32,7 @@ const allScssGlob = join(buildConfig.packagesDir, '**/*.scss');
 const schematicsGlobs = [
   // File templates and schemas are copied as-is from source.
   join(schematicsDir, '**/+(data|files)/**/*'),
-  join(schematicsDir, '**/+(schema|collection).json'),
+  join(schematicsDir, '**/+(schema|collection|migration).json'),
```

Comment:
does this need to be updated as well? https://github.com/angular/material2/blob/master/src/lib/schematics/BUILD.bazel#L19

Reply:
Yep, good catch.